packages/core/src/types.ts

```diff
@@ -19,6 +19,9 @@ export type FormDataConvertible =
   | null
   | undefined
 
+export type HttpMethod = 'get' | 'post' | 'put' | 'patch' | 'delete'
```

Comment:
I wonder whether `VisitMethod` would be better? HTTP methods are usually all caps, and this isn't an exhaustive list of HTTP methods either.

Reply:
Suggestion: you could allow both upper and lowercase: 

```ts
export type HttpVerb = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE'
export type Method = HttpVerb | Lowercase<HttpVerb>
```

And just use `.toLowerCase()` when you work with a user-given method.

Reply:
TIL `Lowercase`!

But wouldn't that break the existing usage of `Method` as an enum in userland code?

Reply:
A bit late, but yeah if users use `Method` themselves for something else than Inertia (eg. they do something like `if (method === 'get')` in their own code, it could break. I'd say that it's unlikely though!